integration-tests/testutils/build/build_test.go

```diff
@@ -263,3 +264,15 @@ func (s *BuildSuite) TestAssetsDoesNotBuildSnapdFromBranchIfNotInstructedTo(c *c
 		check.Commentf("Expected 0 call to execCommand with %s, got %d",
 			buildSnapdCmd, buildCall))
 }
+
+func (s *BuildSuite) TestAssetsHonoursBuildTags(c *check.C) {
+	testBuildTags := "mybuildtag"
+	Assets(s.useSnappyFromBranch, s.arch, testBuildTags)
```

Comment:
Totally agreed thanks! :) All is clearer with the Config parameter, and adding a new field won't force us to change a lot of tests.

Regarding the name of the Assets function, when consumed by a client it is scoped with the name of the package, reading ```build.Assets```, take a look at ```integration-tests/main.go```. Do you think it's ok to leave it like that?